proto/eth/wrapper/beacon_wrapper.proto

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 Prysmatic Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+syntax = "proto3";
+
+package ethereum.eth.wrapper;
+
+import "proto/eth/v2/version.proto";
+import "proto/eth/v1/beacon_state.proto";
+import "proto/eth/v2/beacon_state.proto";
+
+
+// WrappedBeaconState is used only inside db/kv package and it there for the sole purpose of
+// storing the state objects efficiently. With Normalizing the fields in the "state" bucket
+// it also stores some indexes and other metadata fields that cannot be stored in the
+// BeaconState object itself.
+message WrappedBeaconState {
+  v2.Version version = 1001;        // Used to identify the state object
+  bytes validatorIndexes = 1002;    // stores the compressed indexes of the validator entry hashes
```

Comment:
Don't we want to add more data related to the circular buffers(randao mixes, block roots, state roots) ?

Reply:
yes, that would be in another PR. Now its about fixing validator entries.